lib/cypress-on-rails.rb

```diff
@@ -1,3 +1 @@
-require 'cypress_dev'
-
-Cypress = CypressDev
```

Comment:
@grantspeelman would it be enough to have 

```ruby
CypressDev = CypressOnRails
```

In here to maintain backward compatibility?

Reply:
nvm, saw #19 it should pave the way for this and I can then cleanup and rebase this PR